File: CS/MainForm.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Windows.Forms;

namespace DxExample
{
    public partial class MainForm : DevExpress.XtraEditors.XtraForm
    {
        public MainForm()
        {
            InitializeComponent();
            new CategoryView { Dock = DockStyle.Fill, Parent = this }; 
        }
    }
}

File: VB/MainForm.vb
Namespace DxExample
    Partial Public Class MainForm
        Inherits DevExpress.XtraEditors.XtraForm

        Public Sub New()
            InitializeComponent()
            Dim categoryView As CategoryView = New CategoryView With {.Dock = DockStyle.Fill, .Parent = Me}
        End Sub
    End Class
End Namespace
